roles/etcd/templates/etcd.j2

```diff
@@ -64,21 +72,9 @@ start)
     ;;
 
 stop)
-    {% if run_as == "worker" -%}
-    echo "==> no 'stop' action for proxy"
-    {% else -%}
-    #XXX: do better cleanup like remove the member from the cluster only if it was started
-    out=`etcdctl member list | grep {{ node_name }} | awk -F ':' '{print $1}'`
-    if [ "$out" != "" ]; then
-        echo "==> removing member: " $out
-        etcdctl member remove $out
-    fi
-    {% endif -%}
     ;;
 
 post-stop)
-    #XXX: is there a case when we should not cleanup the data dir on stop?
-    rm -rf $ETCD_DATA_DIR
```

Comment:
Is this intentional? My undersatnd was that we need to remove the data-dir on service stop, so just curious if you were seeing any issue with data dir being removed.